diff --git a/cbits/sqlite3.c b/cbits/sqlite3.c
index 66bb609..ca32723 100644
--- a/cbits/sqlite3.c
+++ b/cbits/sqlite3.c
@@ -105731,7 +105731,7 @@ static unsigned int openssl_init_count = 0;
 static void sqlcipher_openssl_log_errors() {
     unsigned long err = 0;
     while((err = ERR_get_error()) != 0) {
-      sqlcipher_log(SQLCIPHER_LOG_ERROR, "sqlcipher_openssl_log_errors: ERR_get_error() returned %lx: %s", err, ERR_error_string(err, NULL));
+      sqlcipher_log(SQLCIPHER_LOG_ERROR, "sqlcipher_openssl_log_errors: ERR_get_error() returned %lx: %s", err, ERR_func_error_string(err));
     }
 }
